_posts/2022-01-11-bouzek-h.md

```diff
@@ -0,0 +1,18 @@
+---
+title: "Heather Bouzek, MS"
+tags:
+  - genomics
+  - phylogeny
+  - bacterial pangenomes
+  - statistics
+  - R
+  - shiny/quarto
+  - little python
+categories: 
+  - mentor
+  - consult
+---
+
+I'm a Bioinformation with the Johnston Lab.  I create data packages and do exploratory and statistical analysis for bacterial pangenomic data.  I'm open for mentoring and consulting.
```

Comment:
Does the Johnston Lab have a website you'd like to link out to here, to do that in markdown it's just [Johnston Lab](https://myurl.com)